services/core-api/app/api/EMLI_contacts/resources/EMLI_contact_list.py

```diff
@@ -0,0 +1,104 @@
+from flask_restplus import Resource
+from werkzeug.exceptions import BadRequest
+from app.extensions import api
+from flask import request
+from app.api.utils.custom_reqparser import CustomReqparser
+from app.api.utils.access_decorators import requires_role_edit_emli_contacts, requires_any_of, VIEW_ALL, MINESPACE_PROPONENT
+from app.api.utils.resources_mixins import UserMixin
+from app.api.EMLI_contacts.models.EMLI_contact import EMLIContact
+from app.api.EMLI_contacts.models.EMLI_contact_type import EMLIContactType
+from app.api.EMLI_contacts.response_models import EMLI_CONTACT_MODEL
+
+
+class EMLIContactListResource(Resource, UserMixin):
+    parser = CustomReqparser()
+    parser.add_argument(
+        'emli_contact_type_code',
+        type=str,
+        trim=True,
+        help='EMLI contact type code',
+        location='json')
+    parser.add_argument(
+        'mine_region_code', type=str, trim=True, help='EMLI mine region code', location='json')
+    parser.add_argument('first_name', type=str, trim=True, help='EMLI First name', location='json')
+    parser.add_argument('last_name', type=str, help='EMLI Last name.', location='json')
```

Comment:
Same as previous comment(services/core-api/app/api/EMLI_contacts/resources/EMLI_contact.py), add trim=True for consistency or ditch it altogther.

Reply:
Added trim=True to last_name